### DISCUSSION TITLE
Unable to run

### DISCUSSION THREAD

**Discussion by pmatthews05** [actions/virtual-environments#9059] [General questions]
I'm brand new to this repo, and I'm just trying to get this to create a Ubuntu 22.04 image. I've forked and downloaded the repo. I've followed the manual instructions in https://github.com/actions/runner-images/blob/main/docs/create-image-and-azure-resources.md, ensured I have the latest packer, az cli etc.

I've attempted to run the script 3 times now directly from my Windows 11 - VSCode laptop and it has failed every time.

```ps
GenerateResourcesAndImage -SubscriptionId "dce4170d-a8be-4e9d-a3ac-c74584bde55f"  `
  -ResourceGroupName "azd1-rg-mgt-dev-ado-agents-01" `
  -AzureLocation "Uk South" `
  -ImageType "Ubuntu2204"
``` 
I've attached the last logs, and I'm not really sure where to go from here. Could someone advise please?

[logs.txt](https://github.com/actions/runner-images/files/13745497/logs.txt)

**Comment by v-sergei-pyshnoi** [comment]
Hello @pmatthews05 . Look like something goes wrong during `install-rust.sh` execution. Unfortunately, we cannot reproduce your behavior. I tried build image using `GenerateResourcesAndImage` from my local environment and everything goes fine. Also our nightly builds are green, so  be sure that you use latest commit from `main` branch.
Possible reason of `script unexpected disconnect` can be on network side . For example it can be some network security polices on your Azure subscription .

**Comment by pmatthews05** [comment]
Hi @sergei-pyshnoi 
 Thank you for have a look at this. Didn't expect a reply on Christmas day, you should have been enjoying your holiday.

I've attempted to run it again, and this time, I kept the laptop plugged in, instead of running from battery. It has completed the build to the end.

Looks like the problem was with me and not the code.
